### ISSUE TITLE
UNKNOWN values of classifier

### ISSUE THREAD

**Issue by riomus** [Affectiva/cpp-sdk-samples#14]
Hi,

I am aware that classification is not an easy task and to have a high level of confidence, you are delaying results and at the beginning. Would it be possible to expose some parameters that would give me the ability to tune classifiers to receive classification values earlier, with the awareness that those results can be not correct?

**Comment by ahamino** [comment]
Are u looking for parameters to tune the confidence window for the appearance metrics ?

**Comment by riomus** [comment]
That is the exact thing we need, we would like to be not get gender/age classification as soon as possible (after K frames), but we will be awere of that , the results can be not correct at some level

**Comment by umangmehta12** [comment]
In the sample apps we [enable all the classifiers for appearance](https://github.com/Affectiva/cpp-sdk-samples/blob/master/opencv-webcam-demo/opencv-webcam-demo.cpp#L119). You can enable only specific classifiers by `frameDetector->setDetectAge(true)`. Please find the list of API to enable disable all/some classifiers (http://developer.affectiva.com/pages/platforms/v3_4_1/windows/classdocs/affdex-native/classaffdex_1_1_detector.html).

**Comment by riomus** [comment]
Yes, I am aware of that but how can we change the certainty window? I am aware that decreasing certainty will decrease accuracy.

**Comment by umangmehta12** [comment]
@riomus Thanks for the feedback. We currently have not exposed the ability to change the certainty window. I will propose this as a feature request to the SDK Team.

**Comment by adennie** [comment]
@riomus (and others who upvoted this issue), could you provide some insight into your use case to help us understand why the current behavior is not meeting your needs?  You should typically be seeing results after a few seconds of input frames, if the SDK is able to determine age/gender with a minimum level of confidence (could be later if it takes some additional time to achieve that confidence).  

Why would a quicker but less accurate result be preferable?

**Comment by riomus** [comment]
@adennie we have som cases where our installation are in places where people are in FOV of camera only for seconds, in those cases, we would like to get results as fast as possible with configurable certainty or any parameter that will give us the ability to control time that we need to wait for results for one person.

**Comment by adennie** [comment]
Thanks, but it's still not clear to me, if in your use case, getting age/gender is important, how useful is it to get a quicker-but-low-accuracy answer?  What level of confidence would be acceptable?

**Comment by riomus** [comment]
So people are in front of camera for seconds or less, we would like to get any information about them, even knowing that the accuracy will be 70% what is still better than random 50% (for 2 class classification), getting any insights about them that are better than random are still very good for us.

**Comment by adennie** [comment]
@riomus thanks for the input.

**Comment by riomus** [comment]
Can you give any update on that issue? Are you planning to add required parameter?

**Comment by riomus** [comment]
Hi, is there any progress on that issue?

**Comment by umangmehta12** [comment]
Upon checking with the team here, we haven't prioritized this at the moment.
